governance/votes/Executive vote - May 30, 2024.md

```diff
@@ -0,0 +1,82 @@
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following actions will occur within the Maker Protocol:
+
+- The Smart Burn Engine (SBE) Surplus Buffer Upper Limit will be updated.
+- A RWAF DAO Resolution will be approved.
+- A RWA004-A DAO Resolution will be approved.
+- A proxy spell for Spark Protocol will be triggered.
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **30 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+### Smart Burn Engine (SBE) Parameter Updates
+
+- **Authorization:** [Forum post](https://forum.makerdao.com/t/smart-burn-engine-vow-hump-surplus-buffer-upper-limit-reconfiguration-update-7/24348/2)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/smart-burn-engine-vow-hump-surplus-buffer-upper-limit-reconfiguration-update-7/24348)
+
+If this executive proposal passes, the following SBE parameter will be changed:
+
+- The [`vow.hump` (SBE Surplus Buffer Upper Limit)](https://mips.makerdao.com/mips/details/MIP104#9-1-smart-burn-engine-parameters) parameter will be increased by 5 million DAI seconds from 50 million DAI to **55 million DAI**.
+
+### Approve RWAF DAO Resolution
+
+- **Authorization:** [Forum post](TBD)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/dao-resolution-banking-setup-for-rwa-foundation/24362)
+
+If this executive proposal passes, the following DAO Resolution pertaining to RWA Foundation (RWAF) will be approved:
+
+- RWAF DAO Resolution with IPFS hash [Qmb8vLDH6wT4Y2axnJX1JSKVKHTG3jzX3U3novw886H8UR](https://ipfs.io/ipfs/Qmb8vLDH6wT4Y2axnJX1JSKVKHTG3jzX3U3novw886H8UR).
+
+### Approve RWA004-A DAO Resolution
+
+- **Authorization:** [Forum post](TBD)
+- **Proposal:** [Forum post](https://forum.makerdao.com/t/harbor-trade-credit-workout-process/24367)
+
+If this executive proposal passes, the following DAO Resolution pertaining to [Harbor Trade Credit (RWA004-A)](https://makerburn.com/#/collateral/RWA004-A) will be approved:
+
+- RWA004-A DAO Resolution with IPFS hash [QmUiYTRy4BkV681tfFe3Ksj6gdLEq27w34MqLw5LvRaBoD](https://ipfs.io/ipfs/QmUiYTRy4BkV681tfFe3Ksj6gdLEq27w34MqLw5LvRaBoD).
+
+### Trigger Spark Proxy Spell
+
+- **Authorization:** [Poll](TBD)
+- **Proposal**: [Forum posts](https://forum.makerdao.com/t/may-21-2024-proposed-changes-to-sparklend-for-upcoming-spell/24327)
+
+If this executive proposal passes, a Spark proxy spell at [TBD](https://etherscan.io/address/TBD) will be triggered. This proxy spell will take the following actions:
```

Comment:
We're working to provide more detailed information about each subDAO element, rather than just listing them. Please check this executive: https://vote.makerdao.com/executive/template-executive-vote-stability-fee-changes-dsr-decrease-march-2024-ad-compensation-q1-2024-avc-member-participation-rewards-aave-sparklend-revenue-share-whitelist-new-address-in-the-rwa015-a-output-conduit-usdp-input-conduit-management-spark-proxy-spell-april-22-2024